src/components/activity/ActivityDirectiveChangelog.svelte

```diff
@@ -112,12 +113,12 @@
     // Manually check remaining fields that could have changed and require extra formatting
 
     if (current.start_offset !== previous.start_offset) {
-      const currentStartTimeDoy = getDoyTimeFromInterval(planStartTimeYmd, current.start_offset);
-      const previousStartTimeDoy = getDoyTimeFromInterval(planStartTimeYmd, previous.start_offset);
+      const currentStartTime = getDoyTimeFromInterval(planStartTimeYmd, current.start_offset);
+      const previousStartTime = getDoyTimeFromInterval(planStartTimeYmd, previous.start_offset);
 
-      differences['Start Time (UTC)'] = {
-        currentValue: currentStartTimeDoy,
-        previousValue: previousStartTimeDoy,
+      differences[`Start Time (${$plugins.time.primary.label})`] = {
```

Comment:
would this potentially get affected by the current time system if it's always using `getDoyTimeFromInterval`?

Reply:
Looks like I missed using the plugin in that case, will add